src/Raven.Server/Utils/ChangeVectorUtils.cs

```diff
@@ -241,14 +242,25 @@ public static string NewChangeVector(string nodeTag, long etag, string dbIdInBas
                 .ToString();
         }
 
-        public static long GetEtagByNodeTag(string changeVector, string nodeTag)
```

Comment:
Did no one used `ByNodeTag` ?

Reply:
This is wrong to get the Etag by the node, since it is not unique and could come from different database or even cluster.